.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,3 +5,11 @@ Fixes #[Fill ID of the Issue]
   - [What did you changed?]
```

Comment:
Using this occasion we could fix grammatical error here ;) `What did you **change**`

Reply:
:)